master/internal/experiment.go

```diff
@@ -923,10 +937,16 @@ func (e *experiment) setWeight(ctx *actor.Context, weight float64) error {
 }
 
 func (e *experiment) setRP(ctx *actor.Context, msg sproto.SetResourcePool) error {
+	// TODO: secure this to prevent people from just changing RPs to whatever they want
```

Comment:
If this is done, we should remove this comment. If not, we should secure before release.

Reply:
this comment shouldn't exist anymore...I'll fix if it still sows up later